amazon/2016.09/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM amazonlinux:2016.09
+
+LABEL maintainer me@danvaida.com
+
+RUN yum update -y \
+ && yum install -y ca-certificates \
```

Comment:
I think only one space is not enough for indentation. I would stick with 4 spaces as used by the other Dockerfiles.